dub.json

```
@@ -13,30 +13,230 @@
         "Sebastián E. Peyrott",
         "John Reimer",
         "Mike Wey",
-        "hauptmech"
+        "hauptmech",
+        "KonstantIMP"
     ],
-    "dependencies": {
-        "gtk-d:gtkd": "*",
-        "gtk-d:gstreamer": "*"
-    },
-
-    "subPackages" : [
-        {
-            "name": "gtkd",
-            "targetType": "library",
-            "targetName": "gtkd-3",
-            "sourcePaths": ["generated/gtkd"],
-            "libs-posix": ["dl"],
-            "importPaths": ["generated/gtkd"]
-        },
-        {
-            "name": "gstreamer",
-            "targetType": "library",
-            "targetName": "gstreamerd-3",
-            "sourcePaths": ["generated/gstreamer"],
-            "libs-posix": ["dl"],
-            "importPaths": ["generated/gstreamer"],
-            "dependencies": {"gtk-d:gtkd":"*"}
-        }
-    ]
+    "subPackages": [
+		{
+			"name": "linker",
+			"targetType": "library",
+			"targetName": "linker",
+			"version" : "1.0.0",
+			"importPaths": ["source/linker/"],
```

style: remove trailing `/`
```suggestion
			"importPaths": ["source/linker"],
```